### PR TITLE
3308b4

### PR DESCRIPTION
**Pull request by john-c-lynch**

### CODE REVIEW

File: protocols/3308b4/3308b4.ot2.apiv2.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    'author': 'Opentrons',
+    'source': 'Custom Protocol Request',
+    'apiLevel': '2.11'   # CHECK IF YOUR API LEVEL HERE IS UP TO DATE
+                         # IN SECTION 5.2 OF THE APIV2 "VERSIONING"
+}
+
+
+def run(ctx):
+
+    [mount_side
```
Comment (ncdiehl11) [inline protocols/3308b4/3308b4.ot2.apiv2.py:13]: please rename `mount_m300` (bear with me as I create the `fields.json` semantics check to our fields variable names

File: protocols/3308b4/3308b4.ot2.apiv2.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    # "True" for park tips, "False" for discard tips
+
+    # load labware
+    spr_well = ctx.load_labware('nest_1_reservoir_195ml', '4')
+    vhb_well = ctx.load_labware('nest_1_reservoir_195ml', '5')
+    water_well = ctx.load_labware('nest_1_reservoir_195ml', '6')
+    spr_plate = ctx.load_labware('nest_96_wellplate_2ml_deep', '7')
+    vhb_plate = ctx.load_labware('nest_96_wellplate_2ml_deep', '8')
+    water_plate = ctx.load_labware('nest_96_wellplate_2ml_deep', '9')
+    '''
```
Comment (ncdiehl11) [inline protocols/3308b4/3308b4.ot2.apiv2.py:30]: we can ditch this multi-line comment

File: protocols/3308b4/3308b4.ot2.apiv2.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    '''
+
+    # load tipracks
+    tiprack = [
+        ctx.load_labware('opentrons_96_tiprack_300ul', '1')
+        ]
+    '''
+
+    Add your tipracks here as a list:
```
Comment (ncdiehl11) [inline protocols/3308b4/3308b4.ot2.apiv2.py:49]: same with this comment

File: protocols/3308b4/3308b4.ot2.apiv2.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    # lists
+    volume_list = [350, 350, 50]
+    source_list = [spr, vhb, water]
+    dest_list = [spr_plate.rows()[0], vhb_plate.rows()[0],
+                 water_plate.rows()[0]]
+
+    # protocol
+    for volume, source, dest in zip(volume_list, source_list, dest_list):
+        m300.pick_up_tip()
+        m300.transfer(volume,
```
Comment (ncdiehl11) [inline protocols/3308b4/3308b4.ot2.apiv2.py:84]: just confirming the intended functionality is to transfer to all wells with the same tip. if so, looks good, just may want to rename the variable `dest` to be plural (`dests`)

File: protocols/3308b4/fields.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "dropDown",
+    "label": "P300-Multi Mount Side",
+    "name": "mount_side",
```
Comment (ncdiehl11) [inline protocols/3308b4/fields.json:5]: please rename `mount_m300`

**Review by ncdiehl11 — CHANGES_REQUESTED** [review]
a few small changes, but overall looking good!

File: protocols/3308b4/3308b4.ot2.apiv2.py (hunk 4 above)
Comment (john-c-lynch) [reply protocols/3308b4/3308b4.ot2.apiv2.py:84]: Yes, each plate uses one set of 8 tips. Will modify

**Pull request merged**